source/MaterialXView/Viewer.cpp

```diff
@@ -952,6 +996,24 @@ void Viewer::drawContents()
     glDisable(GL_CULL_FACE);
     glEnable(GL_FRAMEBUFFER_SRGB);
 
+    if (_drawEnvironment && _envMaterial)
+    {
+        GLShaderPtr envShader = _envMaterial->getShader();
+        auto meshes = _envGeometryHandler->getMeshes();
+        auto envPart = !meshes.empty() ? meshes[0]->getPartition(0) : nullptr;
+        if (envShader && envPart)
+        {
+            glEnable(GL_CULL_FACE);
+            glCullFace(GL_FRONT);
+            envShader->bind();
+            _envMaterial->bindViewInformation(_envMatrix, view, proj);
```

Comment:
Draw with only the env matrix which skips adding in any model scale or translation.